src/app/components/EnterPage/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import styles from './enter.scss';
+import Header from '../NavBar/Header';
+
+let userInfo = [];
+
+class EnterPage extends Component {
+  render() {
+    // get user's information
+    const json = localStorage.getItem('users');
```

Comment:
Take note that we only use `localStorage` as a temporary solution. After the integration with backend, we won't use `localStorage`.